### PR TITLE
Bugfix/sw 1427 fix mr beam plugin unit tests

### PR DESCRIPTION
**Pull request by irlaec**
Most of the tests are now passing, except the ones related to Software Update.
The "frontend" tests were marked as deprecated, as we now do them in Cypress.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,9 +0,0 @@
-###
```
Comment (JFSolution) [inline requirements.txt]: please keep the original file

**Review by JFSolution — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (irlaec) [reply requirements.txt]: Ooops, restored!

File: tests/frontend/critical_designs/base_procedure.py [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 from selenium.webdriver.support import expected_conditions as EC
 
-from frontend import uiUtils
-from frontend import webdriverUtils
-from frontend import gcodeUtils
-from frontend import frontendTestUtils
+from .. import uiUtils
+from .. import webdriverUtils
+from .. import gcodeUtils
+from .. import frontendTestUtils
```
Comment (JFSolution) [inline tests/frontend/critical_designs/base_procedure.py]: I don't like these .. does this work too?

```suggestion
from tests.frontend import webdriverUtils, gcodeUtils, frontendTestUtils, uiUtils
```

**Review by JFSolution — CHANGES_REQUESTED** [review]
please change the from imports to absolute imports

File: tests/frontend/critical_designs/base_procedure.py (hunk 2 above)
Comment (irlaec) [reply tests/frontend/critical_designs/base_procedure.py]: I changed all the relative imports to absolute

**Review by JFSolution — APPROVED** [review]

**Pull request merged**